src/commands/BootstrapCommand.js

```diff
@@ -1,13 +1,12 @@
 import FileSystemUtilities from "../FileSystemUtilities";
-import NpmUtilities from "../NpmUtilities";
 import PackageUtilities from "../PackageUtilities";
+import BootstrapMixin from "../mixins/BootstrapMixin";
 import Command from "../Command";
 import async from "async";
-import find from "lodash.find";
 import path from "path";
 import semver from "semver";
 
-export default class BootstrapCommand extends Command {
+export default class BootstrapCommand extends BootstrapMixin(Command) {
```

Comment:
while this project isn't massive like those at facebook, i agree with their reasoning behind avoiding them:

* Mixins introduce implicit dependencies
* Mixins cause name clashes
* Mixins cause snowballing complexity

i'm sure you've read the post as well: https://facebook.github.io/react/blog/2016/07/13/mixins-considered-harmful.html

IMO, it's best to play it safe. who knows what kind of things we'll be adding down the road. we'd have to refactor some of the functions to accept additional parameters, but i don't think it would be too much work.

another approach may be to extend `Command` with another type of generic command with the added functionality. as an example, something like `FileSystemAccessCommand` which includes (obviously) file system access methods.